### ISSUE TITLE
Make primary selection/cursor stand out in default theme?

### ISSUE THREAD

**Issue by xJonathanLEI** [helix-editor/helix#5281]
I think the default theme can use some improvements regarding selection/cursor colors:

## 1. Primary selection/cursor uses the same color as other selections

Primary selection/cursor vs others are indistinguishable right now. It's impossible to tell which selection is the primary one. Ideally we should make the primary selection stand out with a brighter color.

## 2. Cursor color too jumpy

Currently the cursor color is set to `reversed`. While this is convenient from a theme authoring standpoint, it changes so often that it fails to give a _consistent_ impression on what the primary cursor _should_ look like. I suggest using a fixed color for the primary cursor, and a slightly less bright color for non-primary ones.

Actually, it's impossible to fix issue 1 without getting rid of the use of `reversed` here.

**Comment by archseer** [comment]
Yeah this was discussed here: https://github.com/helix-editor/helix/issues/3842

I was trying to find a good theme setting but couldn't settle on anything. Making it work with reverse will be easier once we land https://github.com/helix-editor/helix/pull/5130

**Comment by xJonathanLEI** [comment]
Ah sorry I'm closing this as a duplicate of #3842 then.
